applications/nrf_desktop/src/modules/config.c

```diff
@@ -20,11 +20,20 @@ LOG_MODULE_REGISTER(MODULE, CONFIG_DESKTOP_CONFIG_LOG_LEVEL);
 
 #define KEY_LEN 11
 
+#if defined(CONFIG_DESKTOP_SETTINGS_LOAD_THREAD_STACK_SIZE)
+#define THREAD_STACK_SIZE	CONFIG_DESKTOP_SETTINGS_LOAD_THREAD_STACK_SIZE
+#else
+#define THREAD_STACK_SIZE	0
```

Comment:
this can be hidden in kconfig
```
if XYZ_ENABLED
config OPT
int "Enable me"
...

else
config OPT
int 0

endif
```
or something similar (we already do it in some places)

Reply:
corrected